feature-changeaddress/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+  id("hedvig.android.ktlint")
+  id("hedvig.android.library")
+  id("hedvig.android.library.compose")
+  @Suppress("DSL_SCOPE_VIOLATION")
+  alias(libs.plugins.serialization)
+}
+
+android {
+  namespace = "com.hedvig.android.feature.changeaddress"
+}
+
+dependencies {
```

Comment:
I see that you added a lot of the things that this module may need but doesn't yet. This is fine, and maybe we can keep it like this while we're working on it, and strip away the ones we're not interested in after this is more or less done.

This can be done by running
`./gradlew :feature-changeaddress:projectHealth` and there's a gradle plugin which will help figure out which dependencies this module doesn't actually need.

Reply:
Oh that is very nice! We can do that later though, I think we will use more of these dependencies during development